dpm/client/__init__.py

```diff
@@ -4,14 +4,41 @@
 from .do_validate import validate
 
 import os
+
 import datapackage
+import requests
+import six
+
 
 class DpmException(Exception):
     pass
 
+class NetworkError(Exception):
```

Comment:
Having only a single exception will give callers a headache to handle properly. On the single base exception - yes, that may be useful.